test/run.dart

```diff
@@ -1,65 +1,374 @@
+import 'package:test/test.dart';
 import 'package:trotter/trotter.dart';
 
 void main() {
-  {
-    print('\nCombinations...');
-    final bagOfItems = characters('abcde'),
-        combos = Combinations(3, bagOfItems);
-    for (final combo in combos()) {
-      print('$combo (${combos.indexOf(combo)})');
-    }
+  final bagOfItems = characters('abcde');
 
-    for (final combo in [
-      ['e', 'c', 'a'],
-      ['a', 'c', 'e']
-    ]) {
-      print('... $combo -> ${combos.indexOf(combo)}');
-    }
-  }
+  group(
+    "Combinations",
+    () {
+      final combos = Combinations(3, bagOfItems);
+      /*
+      for (final combo in combos()) {
+        print('$combo (${combos.indexOf(combo)})');
+      }
+      */
```

Comment:
Remove old code!?

Reply:
This is part of the test that print out the generated list. Since I added `expect()` to verify the same result is given, it does not matter. I just commented them so it can be printed on debug.